cypress/e2e/gui/gui-engage-sphere.spec.cy.js

```diff
@@ -0,0 +1,190 @@
+const API_URL = Cypress.env('API_URL');
+const CUSTOMERS_API_URL = `${API_URL}/customers`;
+describe('Customers - filters and others ', () => {
+    beforeEach(() => {
+        cy.clearCookies();
+        cy.setCookie('cookieConsent', 'accepted');
+        cy.visit('/');
+        cy.contains('h2', 'Hi there!', { timeout: 5000 })
+    });
+
+    it('keeps the filters when coming back from the customer details view', () => {
+        const clientFilterData = {
+            size: 'Small',
+            industry: 'Technology'
+        };
```

Comment:
```suggestion
        };

```

Mesma regra da linha em branco para entender onde acaba algo (neste caso, a definição de uma variável), e onde começa outra (neste caso, os comandos nativos do Cypress).